web/www/horas/Latin/Psalterium/Special/Matutinum Special.txt

```diff
@@ -710,35 +760,76 @@ R. In ætérnum cantábo.
 &Gloria
 R. Misericórdias tuas Dómine * in ætérnum cantábo.
 
-[MM LB6]
+[MM LB6_]
 !Eccl 12:2-3
 v. Bénefac justo, et invénies retributiónem magnam: et si non ab ipso, certe a Dómino. Non est enim ei bene qui assíduus est in malis, et eleemósynas non danti: quóniam et Altíssimus ódio habet peccatóres, et misértus est pœniténtibus.
 $Tu autem
 _
-
 R.br. Dómine, * exáudi oratiónem meam.
 R. Dómine, * exáudi oratiónem meam.
 V. Et clamor meus ad te véniat.
 R. Exáudi oratiónem meam.
 &Gloria
 R. Dómine, * exáudi oratiónem meam.
 
-[MM Capitulum]
+[MM LB6C]
+@:MM LB6_:1-4
+R.br. Misericórdia Dómini * Super timéntes eum.
+R. Misericórdia Dómini * Super timéntes eum.
+V. Ab ætérno et usque in ætérnum.
+R. Super timéntes eum.
+
+[MM LB6]
+@:MM LB6_
+(sed rubrica cisterciensis)
+@:MM LB6C
+
+[MM Capitulum_]
 !1 Cor 16:13-14
 v. Vigiláte, state in fide, viríliter ágite, et confortámini. Ómnia vestra in caritáte fiant.
 $Deo gratias
 _
 V. Beáti qui hábitant in domo tua, Dómine.
 R. In sǽcula sæculórum laudábunt te.
 
-[MM Capitulum Adv]
+[Nocturn 2 Versum Feria6]
+V. Dóminus in Sion magnus.
+R. Et excélsus super omnes pópulos.
+
+[Nocturn 2 Versum Sabbato]
+V. Misit verbum suum, et sanávit eos.
+R. Et erípuit eos de interitiónibus eórum.
+
```

Comment:
@Augustinus-Altovadensis changes in [MM Capitulum] add additional verses ex 6-23-2025 - 1963
probably You could just add `[MM Capitulum] (rubrica cisterciensis)` instead.